src/components/DashBoardNews.spec.js

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import NewsSection from './NewsSection.js';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Header', () => {
+  it('shows a title, logo, headline and text', () => {
+    render(<NewsSection />, { wrapper: MemoryRouter });
+
+    const title = screen.getByText('news', { exact: false });
```

Comment:
You test more specific if you want :)
```suggestion
    const title = screen.getByRole('heading', { level: 2 })
```